snooty/rstspec.toml

```diff
@@ -832,29 +690,28 @@ example = """..cta::
 
```

Comment:
is `cta` the only directive left in the `landing` domain -- if so, what's the rationale behind keeping it in that domain rather than moving it to `mongodb`?

Reply:
Yes! That and `deprecated-version-selector`, which I'm not sure of the importance of (I only see it used [here](https://github.com/mongodb/docs-landing/blob/master/source/legacy.txt#L14)). I'm generally leaning towards getting rid of the landing domain, since it's so sparse

Reply:
Removing `landing` domain...